### ISSUE TITLE
Road to a Minimum Playable Version

### ISSUE THREAD

**Issue by sammy-ette** [OpenSC4-org/OpenSC4#1]
This is a tracking issue that contains some TODOs before releasing some form of beta playable version. Make a comment here to suggest what should be in the **INITIAL release.**

- [ ] #2 
- [ ] God mode
- [ ] Saving

**Comment by Lecrapouille** [comment]
Hi I've just discovered your project. This may give you idea concerning the simulation mode https://github.com/Lecrapouille/OpenGlassBox

**Comment by jacklollz2** [comment]
I agree with the "OpenGlassBox" SimCity 2013 is highly based off of the SimCity 4 system.
